Lekce_02_Ukol01/Kalkulacka.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Lekce_02_Ukol01
+{
+    class Kalkulacka
+    {
+        public double Secti(double a, double b)
+        {
+            double vysledek = a + b;
+            return vysledek;
```

Comment:
nit: dalo by se zkratit
```suggestion
            return  a + b;
```